migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -51,15 +51,15 @@ include::modules/migration-understanding-cam.adoc[leveloffset=+1]
 [id='Configuring-replication-repository_{context}']
 == Configuring a replication repository
 
-You must configure an S3-compatible replication repository. The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[filesystem or snapshot data copy methods].
+You must configure S3 storage as a replication repository. The CAM tool copies data from the source cluster to the replication repository, and from there to the target cluster, using either the xref:migration-understanding-data-copy-methods_{context}[file system or the snapshot data copy method].
```

Comment:
Change to:

You must configure an Object Store to use as a replication repository. The CAM tool copies data from the source cluster to the replication repository, and from there to the target cluster, using either the xref:migration-understanding-data-copy-methods_{context}[file system or the snapshot data copy method].